index.js

```diff
@@ -1,5 +1,9 @@
 const express = require('express')
+const helmet = require('helmet')
+
 const app = express()
+app.use(helmet())
```

Comment:
This looks good. In the future we may want to make sure that we know all the things that helmet is doing and make sure that we don't want to add any additional strictness.